WooCommerce/Classes/ViewModels/OrderDetailsViewModel/OrderDetailsDataSource.swift

```diff
@@ -51,6 +51,10 @@ final class OrderDetailsDataSource: NSObject {
 
     var onCellAction: ((CellActionType, IndexPath?) -> Void)?
 
+    /// Called when table view should reload
+    ///
+    var onReloadTableView: (() -> Void)?
```

Comment:
I would explore removing "tableview" from the name. Maybe `onUIReloadRequired` as in the view model?